runtime/src/node_modules/@sapper/internal/manifest-server.d.ts

```diff
@@ -0,0 +1,29 @@
+export const src_dir: string
+export const build_dir: string
+export const dev: boolean
+export const manifest: Manifest
+
+export type Manifest = {
+  server_routes: ServerRoute[];
+  ignore: RegExp[];
+  root_comp?: {
+    default: SSRComponent<unknown>;
+    preload?: Preload<unknown>;
+  }
+  error: SSRComponent
+  pages: Route[]
+};
+
+export type ServerRoute = {
+  pattern: RegExp;
+  handlers: Record<string, Handler<Req, Res>>;
+  params: (match: RegExpMatchArray) => Record<string, string>;
+};
+
+export interface SSRComponent<Props> {
```

Comment:
Not sure about this semantic-wise, but shouldn't this extend `Component`?

Reply:
Ah, that means Component is too general sounding. I think Component should really be DOMComponent, I just didn't realize that the type would split into two when I was first choosing some of the names.

Reply:
I combined the client and server components into a single `Component`

Reply:
I renamed `Component` to `DOMComponent` since it clarified things a bit to me